src/dso_api/settings.py

```diff
@@ -214,6 +214,7 @@
 
 DATAPUNT_AUTHZ = {
     "JWKS": os.getenv("PUB_JWKS", JWKS_TEST_KEY),
+    "JWKS_URL": os.getenv("KEYCLOAK_JWKS_URL"),
```

Comment:
Kan dit nog problemen geven, als deze env var ontbreekt en er is geen default?

Reply:
Nee, dat is "fallback" en AUTHZ module haald KEYCLOAK allen als deze is gezet.